src/main/java/com/radixdlt/crypto/ECPublicKey.java

```diff
@@ -134,19 +151,11 @@ public String toString() {
 		return String.format("%s[%s]", getClass().getSimpleName(), toBase64());
 	}
 
-	public String toBase64() {
-		return Bytes.toBase64String(this.publicKey);
+	private int computeHashCode() {
```

Comment:
makes more sense for this to be static with an input imo, as it is only really used for construction and should never be called during the lifetime of an instance

Reply:
It probably should, but unfortunately this is the only way that EqualsVerifier will work with cached hashes.  I think this is a small loss of isolation to have all the edge cases that EqualsVerifier covers.